python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -1354,6 +1354,50 @@ keys:
                 override:
                   type: str
                   description: Transceiver type.
+            application_override_100gbase_srbd:
+              type: bool
+              description: |-
+                Set legacy mode for 100GBASE-SRBD interoperability.
+                This option is mutually exclusive with `application_overrides` and `application_override_lanes`, and takes precedence over them.
```

Comment:
let's remove this and just have it as valid value on the `application override`, which then needs to be a string with valid values `"0", "1", "2" .... "15"` and `100gbase_srbd`